modules/stanford_intranet/lib/scss/base/intranet.scss

```diff
@@ -1 +1,34 @@
 @charset "UTF-8";
+@import "./stanford_intranet.config";
```

Comment:
is the changes in this file intentional for a PR for the publication teasers??

Reply:
That is unintentional and should not be here. My mistake.